README.md

```diff
@@ -2,39 +2,298 @@
-| `/field/application/:client-id/context/:api-context/version/:api-version` | `DELETE` | Deletes the field values of an API subscription                                                | `curl -v -X DELETE "http://localhost:9650/field/application/hBnFo14C0y4SckYUbcoL2PbFA40a/context/ciao-api/version/1.0" -H "Cache-Control: no-cache"`                                                                                                                                         | 404 NOT_FOUND / 204 No Content                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+## PUT Field Definitions 
+### `PUT /definition/context/:apiContext/version/:apiVersion`
+Creates or updates the definitions of the subscriptions fields for an API.
 
+### Response with
 
-## Tests
-Some tests require MongoDB to run. 
-Thus, remember to start up MongoDB if you want to run the tests locally.
-There are unit tests, integration tests and acceptance tests plus code coverage reports are generated too.
-In order to run them, use this command line:
+| Status | Description                          |
+|--------|--------------------------------------|
+| 201    | Created                              |
+| 200    | Updated                              |
+| 400    | Bad request                          |
+| 404    | Not found                            |
+
+### PUT Field Definitions example
+
+#### CURL command*
+```
+curl -v -X PUT "http://localhost:9650/definition/context/hello/version/1.0" -H "Cache-Control: no-cache" -H "Content-Type: application/json" -d '{ "fieldDefinitions": [ { "name": "callback-url", "description": "Callback URL", "type": "URL" }, { "name": "token", "description": "Secure Token", "type": "SecureToken" } ] }' 
```

Comment:
What do you think if we spit the curl commands in multiple lines?
It would be easier to read.
For example:
```sh
curl \
    -v -X PUT \
    "http://localhost:9650/definition/context/hello/version/1.0" \
    -H "Cache-Control: no-cache" \
    -H "Content-Type: application/json" \
    -d '{ "fieldDefinitions": [ { "name": "callback-url", "description": "Callback URL", "type": "URL" }, { "name": "token", "description": "Secure Token", "type": "SecureToken" } ] }'
```

Reply:
I think this is not worthwhile